src/background/offline.js

```diff
@@ -41,6 +42,10 @@ export class OfflineManager extends Component {
       await this.sendMessage("onlineDetected");
     } catch (e) {
       log("We are still offline");
+
+      // Duplicate the timeout for the next round.
+      this.timeout *= 2;
```

Comment:
Do we need to reset `this.timeout` after a successful connection? Or is it possible that this value may just keep growing over the days/months until it's exceedingly large?

Reply:
This is reset at the next setProxyState, but I enforce a reset here as well.